README.md

```diff
@@ -1,8 +1,251 @@
+  export GITEA_PASSWORD=$(kubectl get Secret -n fuseml-workloads gitea-creds -o jsonpath="{.data.password}" | base64 -d)
+  ```
+  It is possible to use external gitea server instead. Make sure to provide correct environment variables.
+  
+  `TEKTON_DASHBOARD_URL` is the path to the Tekton server. As with other components, Tekton is installed by `fuseml-installer` into your cluster. To get the right URL, call
+  ```bash
+  export TEKTON_DASHBOARD_URL=$(kubectl get VirtualService -n tekton-pipelines tekton -o jsonpath="{.spec.hosts[0]}")
+  ```
+
+  Now it's possible to execute `bin/fuseml_core`.
+  Use the `--help` flag to get the command line options that you can supply. By default the server listens on the follwing ports: 8000 (http) and 8080 (grpc)
+
+* Run the client
+
+  Executing the client with `--help` option will show the usage instuctions
+```bash
+    > bin/fuseml_core-cli --help
+bin/fuseml_core-cli is a command line client for the FuseML API.
+
+Usage:
+    fuseml_core-cli [-host HOST][-url URL][-timeout SECONDS][-verbose|-v] SERVICE ENDPOINT [flags]
+
+    -host HOST:  server host (dev). valid values: dev, prod
+    -url URL:    specify service URL overriding host URL (http://localhost:8080)
+    -timeout:    maximum number of seconds to wait for response (30)
+    -verbose|-v: print request and response details (false)
+
+Commands:
+    application (list|register|get|delete)
+    codeset (list|register|get|delete)
+    runnable (list|register|get)
+    workflow (list|register|get|assign|list-runs)
+
+Additional help:
+    bin/fuseml_core-cli SERVICE [ENDPOINT] --help
+
+Example:
+    bin/fuseml_core-cli application list --type "predictor" --workflow "mlflow-sklearn-e2e"
+    bin/fuseml_core-cli codeset list --project "mlflow-project-01" --label "mlflow"
+    bin/fuseml_core-cli runnable list --id "ml-trainer-123" --kind "trainer" --labels '{
+          "function": "predict|train",
+          "library": "pytorch"
+       }'
+    bin/fuseml_core-cli workflow list --name "workflow"
+```
+
+  Without `-url` argument, client tries to connect to the server running locally at your machine. To find the URL of the fuseml-core server service that is installed in your Kubernetes cluster, call 
+
+  ```bash
+  kubectl get VirtualService -n fuseml-core fuseml-core -o jsonpath="{.spec.hosts[0]}"
+  ```
+  and pass the value to each client call with `-url` option.
+
+
+  From the usage output you can see that with the client you can operate with various FuseML artefacts.
+
+  * Codesets describe the code of your ML application, for example MLflow project. They are currently implemented as git repositories.
+
+    Create new codeset by calling `bin/fuseml_core-cli codeset register` command. use the `--help` option to get the description of the command usage.
+
+    Example:
+    ```bash
+    bin/fuseml_core-cli codeset register -name "test" --project "mlflow-project-01" --location "/tmp/mlflow/mlflow-01"
```

Comment:
why some parameter are passed with `-` and others with `--`?

Reply:
For some reason, it is supported by goa ... but you are right it should be consistent in the docs...